src/components/LeaguesTabItem/LeaguesTabItemStyle.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const LeaguesTabItemStyle = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  > * {
```

Comment:
`> *` bad solution, in the future it can make it difficult to support, I mean when something changes it will make difficult to find why it doesn't work correctly.